setup.py

```diff
@@ -22,6 +22,7 @@
       long_description=long_description,
       author='Andreas Kloeckner',
       author_email='inform@tiker.net',
+      python_requires="~=3.6",
```

Comment:
What is `~=`?

Reply:
https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

Reply:
OK. Seems like an odd thing considering there will never be a Python 4 ;)